tests/api/driver_core.c

```diff
@@ -24,12 +24,11 @@ status_t prepare_transfer(const iota_config_t* const iconf, const iota_client_se
     goto done;
   }
 
-  tryte_t msg_tryte[NUM_TRYTES_SERIALIZED_TRANSACTION];
-  flex_trits_to_trytes(msg_tryte, req->msg_len / 3, req->message, req->msg_len, req->msg_len);
+  transfer_t transfer = {.value = 0, .timestamp = current_timestamp_ms(), .msg_len = req->msg_len};
 
-  transfer_t transfer = {.value = 0, .timestamp = current_timestamp_ms(), .msg_len = req->msg_len / 3};
-
-  if (transfer_message_set_trytes(&transfer, msg_tryte, transfer.msg_len) != RC_OK) {
+  tryte_t message_trytes[NUM_TRYTES_MESSAGE] = {};
```

Comment:
Is this array necessary?

Reply:
same as here https://github.com/DLTcollab/tangle-accelerator/pull/587#discussion_r417772434